src/Process/3dTilesProcessing.js

```diff
@@ -222,20 +222,25 @@ export function pre3dTilesUpdate(context, layer) {
 
 const cameraLocalPosition = new THREE.Vector3();
 const worldPosition = new THREE.Vector3();
-function computeNodeSSE(camera, node) {
+const boundingVolumeBox = new THREE.Box3();
+const boundingVolumeSphere = new THREE.Sphere();
+export function computeNodeSSE(camera, node) {
     node.distance = 0;
     if (node.boundingVolume.region) {
         worldPosition.setFromMatrixPosition(node.boundingVolume.region.matrixWorld);
         cameraLocalPosition.copy(camera.camera3D.position).sub(worldPosition);
         node.distance = node.boundingVolume.region.box3D.distanceToPoint(cameraLocalPosition);
     } else if (node.boundingVolume.box) {
-        worldPosition.setFromMatrixPosition(node.matrixWorld);
-        cameraLocalPosition.copy(camera.camera3D.position).sub(worldPosition);
-        node.distance = node.boundingVolume.box.distanceToPoint(cameraLocalPosition);
+        // boundingVolume.box is affected by matrixWorld
+        boundingVolumeBox.copy(node.boundingVolume.box);
+        boundingVolumeBox.applyMatrix4(node.matrixWorld);
+        node.distance = boundingVolumeBox.distanceToPoint(camera.camera3D.position);
     } else if (node.boundingVolume.sphere) {
-        worldPosition.setFromMatrixPosition(node.matrixWorld);
-        cameraLocalPosition.copy(camera.camera3D.position).sub(worldPosition);
-        node.distance = Math.max(0.0, node.boundingVolume.sphere.distanceToPoint(cameraLocalPosition));
+        // boundingVolume.sphere is affected by matrixWorld
+        boundingVolumeSphere.copy(node.boundingVolume.sphere);
+        boundingVolumeSphere.applyMatrix4(node.matrixWorld);
+        node.distance = Math.max(0.0,
+            boundingVolumeSphere.distanceToPoint(camera.camera3D.position));
```

Comment:
Relying on `camera.camera3D.position` assumes that the camera's `matrix` is equal to its `matrixWorld` (as when the camera is a direct child of the scene). I would rather not have this assumption but if we have, let's write it somewhere.

To be conservative, but correct, I think you should apply `node.matrixWorld * camera.matrixWorldInverse` instead and measure the distance to the origin.

Reply:
Agreed. As there are other places that make the same assumption I suggest to not fix  this in this PR.

I can add a // TODO to not forget about this?

Reply:
it would seem wise:
* to create an issue, to specify the problem and indicate where are the other places that make the same assumption (because apparently, this can be a source of bugs)
* apply the solution here and propose it also in the same issue

Reply:
> apply the solution here and propose it also in the same issue

It's not free performance-wise so I won't sneak this small change in an unrelated PR.

I've opened the following issue to not forget about it: https://github.com/iTowns/itowns/issues/800

Reply:
> I can add a // TODO to not forget about this?

just add the TODO